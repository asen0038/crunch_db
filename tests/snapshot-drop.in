set a 1 2 3
set b 4 5 6
snapshot
drop 1
rollback 1
list entries
bye
