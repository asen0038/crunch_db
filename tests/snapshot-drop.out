>
ok

>
ok

>
saved as snapshot 1

>
ok

>
no such snapshot

>
b [4 5 6]
a [1 2 3]

>
bye
